_includes/manuals/nightly/1.2_release_notes.md

```diff
@@ -4,6 +4,10 @@ This section will be updated prior to the next release.
 
 ### Headline features
 
+#### External IPAM
+
+New Smart Proxy External IPAM API allows implementation of IPAM providers. Subnets can be now associated with IPAM plugins providing IP address management capabilities via external entities. An experimental implementaion for phpIPAM is available as a Smart Proxy plugin.
```

Comment:
Please add a link for users interested in more information, eg to the discourse discussion about it